AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -105,4 +122,140 @@ - (void)testTransmissionTargetForToken {
       [MSAnalytics trackEvent:event3 withProperties:properties forTransmissionTarget:childTransmissionTarget3]));
 }
 
+- (void)testTransmissionTargetEnabledState {
```

Comment:
It is probably worth doing a test with a long chain of children and a test with a large number of first-level children.  And we should include a test for the case @guperrot found where parts of a tree haven't been stored locally yet.

Reply:
OK I'll do the test with the long chain of children and a large number of first level children. The other test is already covered by the later addition of "childTarget2-guid" into the tree.